kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -353,7 +353,121 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
   readActivity: 'Read',
   watchActivity: 'Watch',
   exploreActivity: 'Explore',
-  topicLabel: 'Topic',
+
+  folder: 'Folder',
+
+  // Library Categories
+  school: 'School',
+  basicSkills: 'Basic skills',
+  work: 'Work',
+  dailyLife: 'Daily life',
+  forTeachers: 'For teachers',
+
+  // School Categories
+  mathematics: 'Mathematics',
+  sciences: 'Sciences',
+  literature: 'Literature',
+  socialSciences: 'Social Sciences',
+  arts: 'Arts',
+  computerScience: 'Computer Science',
+  languageLearning: 'Language learning',
+  history: 'History',
+
+  // Mathematics Subcategories
+  arithmetic: 'Arithmetic',
+  algebra: 'Algebra',
+  geometry: 'Geometry',
+  calculus: 'Calculus',
+  statistics: 'Statistics',
+
+  // Sciences Subcategories
+  biology: 'Biology',
+  chemistry: 'Chemistry',
+  physics: 'Physics',
+  earthScience: 'Earth science',
+  astronomy: 'Astronomy',
+
+  //  Literature Subcategories
+  readingComprehension: 'Reading comprehension',
+  writing: 'Writing',
+  logicAndCritialThinking: 'Logic and critical thinking',
+
+  // Social Sciences Subcategories
+  politicalScience: 'Political science',
+  sociology: 'Sociology',
+  anthropology: 'Anthropology',
+  civicEducation: 'Civic education',
+
+  // Arts Subcategories = {
+  visualArt: 'Visual art',
+  music: 'Music',
+  dance: 'Dance',
+  drama: 'Drama',
+
+  //  Computer Science Subcategories
+  programming: 'Programming',
+  mechanicalEngineering: 'Mechanical engineering',
+  webDesign: 'Web design',
+
+  // Basic Skills
+  literacy: 'Literacy',
+  numeracy: 'Numeracy',
+  digitialLiteracy: 'Digital literacy',
+  logicalAndCriticalThinking: 'Logic and critical thinking',
+  learningSkills: 'Learning skills',
+
+  // Work Categories
+  professionalSkills: 'Professional skills',
+  technicalAndVocationalTraining: 'Technical and vocational training',
+
+  //  VocationalSubcategories
+  toolsAndSoftwareTraining: 'Tools and software training',
+  skillsTraining: 'Skills training',
+  industryAndSectorSpecific: 'Industry and sector specific',
+
+  // Daily Life Categories
+  publicHealth: 'Public health',
+  entrepreneurship: 'Entrepreneurship',
+  financialLiteracy: 'Financial literacy',
+  currentEvents: 'Current events',
+  environment: 'Environment',
+  mediaLiteracy: 'Media literacy',
+  diversity: 'Diversity',
+  mentalHealth: 'Mental health',
+
+  // Teacher-Specific Categories
+  guides: 'Guides',
+  lessonPlans: 'Lesson Plans',
```

Comment:
Switch to `Lesson plans` to be consistent with the sentence case in all the other categories.